tests/main/interfaces-system-observe/task.yaml

```diff
@@ -1,5 +1,8 @@
 summary: Ensures that the system-observe interface works.
 
+environment:
+    DISPLAY: :0
```

Comment:
I'm surprised this is needed for working with the system bus. Why is it needed? You aren't using dbus-launch anywhere that I can see....